modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradePortletDisplayCheck.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.source.formatter.check;
+
+import com.liferay.petra.string.StringUtil;
+import com.liferay.source.formatter.check.util.JavaSourceUtil;
+import com.liferay.source.formatter.parser.JavaClass;
+import com.liferay.source.formatter.parser.JavaClassParser;
+import com.liferay.source.formatter.parser.JavaMethod;
+import com.liferay.source.formatter.parser.JavaTerm;
+
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradePortletDisplayCheck extends BaseUpgradeCheck {
+
+	@Override
+	protected String format(
+			String fileName, String absolutePath, String content)
+		throws Exception {
+
+		String newContent = content;
+
+		if (fileName.endsWith(".jsp")) {
+			Matcher getPortletInstanceConfigurationMatcher =
+				_getPortletInstanceConfigurationPattern.matcher(content);
+
+			while (getPortletInstanceConfigurationMatcher.find()) {
+				return _getNewContent(
+					newContent, fileName,
```

Comment:
@TamyrisTorres, shouldn't the first parameter be `content` instead of `newContent`?

Reply:
In this case, I chose the 'content' variable for representation 'content or javaMethodContent' variable. 
Maybe, I can change to 'dynamicContent'. It's good for you?
Because this variable can have the content of the test file or just the content of the method java.

Reply:
Hey @TamyrisTorres, sorry if I wasn't clear in my previous comment. :smile: 

My point is why did you pass as first parameter `newContent` instead of `content` only? No need to change the parameter name.

Is it clearer?